cgroups/systemd/apply_systemd.go

```diff
@@ -212,6 +213,17 @@ func (m *Manager) Apply(pid int) error {
 
 	m.Paths = paths
 
+	cpuShares, err := ioutil.ReadFile(path.Join(m.Paths["cpu"], "cpu.shares"))
+	if err != nil {
+		return err
+	}
+
+	if c.CpuShares != 0 {
+		if strings.TrimSpace(strconv.FormatInt(c.CpuShares, 10)) != strings.TrimSpace(string(cpuShares)) {
```

Comment:
I think it's better to use `fmt.Sprintf("%d", c.CpuShares)`, also you don't need `TrimSpace` for this.
But probably it's better to `Fscanf` int directly from file. I'm not sure if it makes sense for one value, but sometimes you need to wrap opened file to `BufReader` for `Fscanf`